utils/src/test/kotlin/FileMatcherTest.kt

```diff
@@ -50,6 +50,18 @@ class FileMatcherTest : WordSpec({
                 defaultMatcher.matches(it) shouldBe true
             }
         }
+
+        "match commonly used license file paths in lower-case" {
+            COMMONLY_USED_LICENSE_FILE_NAMES.forAll {
```

Comment:
I believe we either should add an explicit `toLowerCase()` mapping here, or add a code comment to `COMMONLY_USED_LICENSE_FILE_NAMES ` that all entries need to be lower case by convention.

Reply:
Done.